docs/docs/rest-api/public/api/v2/types/pod.raml

```diff
@@ -1,150 +1,205 @@
+              acceptedResourceRoles?:
+                type: string[]
+                example: [public-facing]
+                description: |
+                  A list of resource roles.
+                  Marathon considers only resource offers with roles in this list for
+                  launching tasks of this app. If you do not specify this,
+                  Marathon considers all resource offers with roles that have been
+                  configured by the `--default_accepted_resource_roles` command line flag.
+                  If no `--default_accepted_resource_roles` was given on startup,
+                  Marathon considers all resource offers. To register Marathon for a role,
+                  you need to specify the `--mesos_role` command line flag on startup.
+                  If you want to assign all resources of a slave to a role,
+                  you can use the `--default_role` argument when starting up the slave.
+                  If you need a more fine-grained configuration, you can use the
+                  `--resources` argument to specify resource shares per role.
+                  See [the Mesos attribute and resources documentation](http://mesos.apache.org/documentation/latest/attributes-resources/) for details
+  PodStatus:
+    (scalaType): PodStatus
+    description: |
+      Pod status communicates the lifecycle phase of the pod, current instance and container
+      status, and recent termination status history.
+    additionalProperties: false
+    properties:
+      id:
+        type: !include pathId.raml
+        description: |
+          Fully qualified name of the pod that this status object is describing.
+      spec:
+        type: Pod
+        description: |
+          The latest version of the pod specification (that has the same pod ID).
+      status:
+        type: string
+        (scalaType): PodState
+        enum: [ DEGRADED, STABLE, TERMINAL ]
+        description: |
+          DEGRADED - The number of STABLE pod instances is less than the number of desired instances.
+          STABLE   - All launched pod instances have started and, if health checks were specified, are all healthy.
+          TERMINAL - Marathon is tearing down all of the instances for this pod.
+      statusSince:
+        type: datetime
+        description: |
+          Time at which the status code was last modified.
+      message?:
+        type: string
+        description: |
+          Human-friendly explanation for reason of the current status.
+      instances?:
+        type: array
+        items:
+          (scalaType): PodInstanceStatus
+          type: !include pod-instance-status.raml
+      terminationHistory?:
+        type: array
+        description: |
+          List of most recent instance terminations.
+          TODO(jdef) determine how many items might show up here; current thinking is .. not many
```

Comment:
Keeping only the most recent failure seems rather ... limiting, but OK we could have a history of 1 here.

For those following along, TaskFailure has this useful information:
```
case class TaskFailure(
  appId: PathId,
  taskId: mesos.TaskID,
  state: mesos.TaskState,
  message: String = "",
  host: String = "",
  version: Timestamp = Timestamp.now,
  timestamp: Timestamp = Timestamp.now,
  slaveId: Option[mesos.SlaveID] = None)
```